apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/GrunnlagsendringshendelseService.kt

```diff
@@ -77,16 +79,18 @@ class GrunnlagsendringshendelseService(
     fun oppdaterAdressebeskyttelseHendelse(
         adressebeskyttelse: Adressebeskyttelse
     ) {
+        val gradering = adressebeskyttelse.adressebeskyttelseGradering
         val sakIder = finnSakIdForFnr(adressebeskyttelse.fnr).distinct()
         sakIder.forEach { sakId ->
-            val oppdaterAdressebeskyttelse = sakServiceAdressebeskyttelse.oppdaterAdressebeskyttelse(
+            sakServiceAdressebeskyttelse.oppdaterAdressebeskyttelse(
                 sakId,
-                adressebeskyttelse.adressebeskyttelseGradering
-            )
-            logger.info(
```

Comment:
Fjernet denne da det ikke virket greit å logge dette til vanlig logg? Åpen for innspill.

Reply:
Vet ikke 100% siden det er bare sakId - men det føles tryggere å flytte det til sikkerLogg.